.gitlab-ci.yml

```diff
@@ -141,7 +142,7 @@ test:rocm_install:
   script:
     - mkdir build_only_install
     - cd build_only_install
-    - CXX=hcc cmake -DBUILD_TEST=OFF ../.
+    - CXX=hcc cmake -DBUILD_TEST=OFF -DROCM_EXPORT_PARSE_TARGETS_LIBRARY=ON ../.
```

Comment:
variable names are different

Reply:
PYTORCH_FOUND_HIP

Reply:
I will update this PR based on the rocThrust, but first please check rocThrust PR:
https://github.com/ROCmSoftwarePlatform/rocThrust/pull/91